sugoi-api-core/src/main/java/fr/insee/sugoi/core/service/impl/UserServiceImpl.java

```diff
@@ -262,6 +262,8 @@ public void delete(String realmName, String storage, String id) {
                   });
           result.getResults().addAll(temResult.getResults());
           result.setTotalElements(result.getResults().size());
```

Comment:
here the total elements is the number of result in the page while with a storage it is all the element that can be found